cmd/brew-test-bot.rb

```diff
@@ -710,6 +710,11 @@ def homebrew
       return if @skip_homebrew
 
       if !@tap && (@formulae.empty? || @test_default_formula)
+        test "brew", "man"
+        test "git", "diff", "--exit-code", "-C", HOMEBREW_REPOSITORY, "docs/brew.1.html"
+        test "git", "diff", "--exit-code", "-C", HOMEBREW_REPOSITORY, "manpages/brew.1"
+        test "git", "diff", "--exit-code", "-C", HOMEBREW_REPOSITORY, "manpages/brew-cask.1"
```

Comment:
added in https://github.com/Homebrew/brew/pull/1177, I'll update this PR accordingly...